code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -55,6 +57,41 @@
 /mob/living/simple_animal/hostile/Life()
 	if(timestopped)
 		return 0 //under effects of time magick
+	if(!client || deny_client_move) //Ventcrawling stuff
+		if(entry_vent)
+			if(get_dist(src, entry_vent) <= 1)
```

Comment:
use `Adjacent(entry_vent)` instead, because this would let you ventcrawl into a vent covered by glass or a wall for example

Reply:
Done